src/components/ContentOpenWith/IntegrationPortal.js

```diff
@@ -25,6 +27,7 @@ class IntegrationPortal extends PureComponent<Props> {
     constructor(props: Props) {
         super(props);
 
+        this.document = document;
```

Comment:
Consider moving this to `componentDidMount`, since it doesn't seem related to the portal creation.